src/index.html

```diff
@@ -22,6 +22,370 @@
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              href="#"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <article
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__img-container">
+          <img
+            src="images/imac.jpeg"
+            class="card__img"
+            alt="imac foto"
+          />
+        </div>
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <section class="card__price">
+          <div class="card__price-block">Price:</div>
+          <div class="card__price-money">$2,199</div>
+        </section>
+        <button
+          class="card__button"
+          data-qa="hover card-hover"
+        >
+          Buy
+        </button>
+      </article>
+
+      <article class="card">
+        <div class="card__img-container">
+          <img
+            src="images/imac.jpeg"
+            class="card__img"
+            alt="imac foto"
```

Comment:
The 'src' attribute for images is repeated multiple times with the same value. If the images are the same for each card, consider using a CSS background-image for the '.card__img-container' class instead to reduce HTML code duplication.